libraries/shared/watcher/storage_watcher_test.go

```diff
@@ -368,15 +379,32 @@ func SharedExecuteBehavior(input *ExecuteInput) {
 				Expect(mockDiffsRepository.MarkNoncanonicalPassedID).To(Equal(fakePersistedDiff.ID))
 			})
 
-			It("does not mark diff as 'noncanonical' if block height is within reorg window", func() {
+			It("marks diff as 'pending' if block height is within reorg window - because we aren't sure its noncanonical yet", func() {
```

Comment:
The term "pending" seems like it could be clearer. What is actually "pending" for a diff in this state? A re-check? If we can fill in the blank in "diffs in the 'pending' state are waiting for ________" it might point to a clearer name.